src/controller.py

```diff
@@ -225,14 +225,18 @@ def analyze_model(self):
         self.load_stl(self.model.opened_stl)
 
     def colorize_model(self):
-        self.save_settings("vip")
+        try:
+            self.save_settings("vip")
+
+            s = sett()
+            shutil.copyfile(s.slicing.stl_file, s.colorizer.copy_stl_file)
+            save_splanes_to_file(self.model.splanes, s.slicing.splanes_file)
+            call_command(s.colorizer.cmd)
+            self.load_stl(s.colorizer.copy_stl_file, colorize=True)
+            self.model.opened_stl = s.slicing.stl_file
+        except IOError as e:
```

Comment:
Are you sure about exception class? If you catch only _IOError_ aren't there any other types?
Or if you want to specifically check that you copied using _shutil_ maybe wrap only it.